bicep/main.bicep

```diff
@@ -198,6 +198,15 @@ module dnsZone './dnsZone.bicep' = if (!empty(dnsZoneId)) {
 @description('Installs the AKS KV CSI provider')
 param azureKeyvaultSecretsProvider bool = false
 
+@description('Rotation poll interval for the AKS KV CSI provider')
+@allowed([
+  '2m'
+  '5m'
+  '10m'
+  '30m'
+])
+param kvPollInterval string = '2m'
```

Comment:
I think having this parameter as a integer would be good.
You can then do the string translation further down where this parameter is referenced.

```bash
rotationPollInterval: '${kvPollInterval}m'
```

Reply:
Scrap this idea. You can pass `30s` as a value.  Keeping it as a string is perfect